code/modules/mining/abandoned_crates.dm

```diff
@@ -103,6 +103,7 @@
 
 /obj/structure/closet/crate/secure/loot/emag_act(mob/user)
 	if(locked)
+		balloon_alert(user, "anti-tamper system activated") //idiot
```

Comment:
Do we need this? The feedback is provided in the uh...y'know...explosion.

Reply:
Guess it doesn't matter too much, but eh.

Reply:
my thought process was that all these emag interactions have a balloon alert so it was there for consistency

and yeah it's harmless